### PR TITLE
DT-24458: Update the list of maintainers

### PR DESCRIPTION
**Pull request by ryanjackson-wf**
# [DT-24458](https://jira.atl.workiva.net/browse/DT-24458)
Issue Status

Add myself as a maintainer of this repo.

### CODE REVIEW

**Comment by aviary2-wf** [comment]
## Security Insights
No security relevant content was detected by automated scans.

## Action Items
* Review PR for [security impact](https://wiki.atl.workiva.net/display/SECURITY/Development+Security+Review+Guidelines); comment "security review required" if needed or unsure
* Verify [`aviary.yaml`](https://wiki.atl.workiva.net/pages/viewpage.action?pageId=42762752#SecurityInsightFramework&Tooling-ConfiguringAviary) coverage of security relevant code
---
_Questions or Comments? Reach out on Slack: #support-infosec._

File: README.md [hunk 1]
```diff
@@ -150,7 +149,7 @@ interface and the most expensive operation in CPU profiling is the interface
 method which in turn calls into runtime.assertI2T.  We need generics.
 
 #### Immutable B Tree
-A btree based on two principals, immutablability and concurrency. 
+A btree based on two principals, immutability and concurrency.
```
Comment (alexandercampbell-wk) [inline README.md:152]: s/principals/principles
:)

**Review by alexandercampbell-wk — DISMISSED** [review]

File: README.md (hunk 1 above)
Comment (ryanjackson-wf) [reply README.md:152]: Good catch. I missed that. I will correct it as well.

Comment (alexandercampbell-wk) [reply README.md:152]: I'll re-approve immediately.

**Comment by alexandercampbell-wk** [comment]
QA +1

**Review by rmconsole-wf — APPROVED** [review]
+1 from RM

File: README.md [hunk 2]
```diff
@@ -150,7 +149,7 @@ interface and the most expensive operation in CPU profiling is the interface
 method which in turn calls into runtime.assertI2T.  We need generics.
 
 #### Immutable B Tree
-A btree based on two principals, immutablability and concurrency.
```
Comment (alsfranyrjb2-ops) [inline README.md:153]: F[]()[]()_**@aviary2-wf 
**_

Comment (alsfranyrjb2-ops) [reply README.md:153]: **_[]()@ryanjackson-wf _**

File: README.md [hunk 3]
```diff
@@ -229,6 +228,6 @@ Requirements to commit here:
 
 
 ### Maintainers
-
- - Dustin Hiatt <[dustin.hiatt@workiva.com](mailto:dustin.hiatt@workiva.com)>
```
Comment (alsfranyrjb2-ops) [inline README.md:233]: _**[]()

**_

File: README.md (hunk 1 above)
Comment (alsfranyrjb2-ops) [reply README.md]: **_[]()

_**

Comment (alsfranyrjb2-ops) [reply README.md]: @

**Comment by alsfranyrjb2-ops** [comment]
> +1 from RM

**Pull request merged**